### PR TITLE
Sync with the English version & minor tweaks

### PR DESCRIPTION
**Pull request by haiiiiiyun**

### CODE REVIEW

File: zh-cn/yaml-cn.html.markdown [hunk 1]
```diff
@@ -87,50 +96,75 @@ a_sequence:
...
+# 像 Python 一样，集合仅是值为 null 的映射；上面的集合等价于：
 set2:
     item1: null
     item2: null
     item3: null
 ```
+
+### 更多相关资源
+
++ [YAML 官方网站](http://yaml.org/)
```
Comment (vendethiel) [inline zh-cn/yaml-cn.html.markdown]: Please add a (EN) tag or change the link for a chinese version

**Comment by haiiiiiyun** [comment]
No Chinese version links found, so I just leave the tags untouched.

**Comment by vendethiel** [comment]
leaving it @g-liu now :)

**Comment by geoffliu** [comment]
Nice, thanks!

**Pull request merged**